libr/core/cmd_anal.c

```diff
@@ -724,41 +724,41 @@ static const char *help_msg_ax[] = {
 	NULL
 };
 
-static void cmd_anal_init(RCore *core) {
-	DEFINE_CMD_DESCRIPTOR (core, a);
-	DEFINE_CMD_DESCRIPTOR (core, aa);
-	DEFINE_CMD_DESCRIPTOR (core, aar);
-	DEFINE_CMD_DESCRIPTOR (core, ab);
-	DEFINE_CMD_DESCRIPTOR (core, ac);
-	DEFINE_CMD_DESCRIPTOR (core, ad);
-	DEFINE_CMD_DESCRIPTOR (core, ae);
-	DEFINE_CMD_DESCRIPTOR (core, aea);
-	DEFINE_CMD_DESCRIPTOR (core, aec);
-	DEFINE_CMD_DESCRIPTOR (core, aep);
-	DEFINE_CMD_DESCRIPTOR (core, af);
-	DEFINE_CMD_DESCRIPTOR (core, afb);
-	DEFINE_CMD_DESCRIPTOR (core, afc);
-	DEFINE_CMD_DESCRIPTOR (core, afC);
-	DEFINE_CMD_DESCRIPTOR (core, afi);
-	DEFINE_CMD_DESCRIPTOR (core, afl);
-	DEFINE_CMD_DESCRIPTOR (core, afll);
-	DEFINE_CMD_DESCRIPTOR (core, afn);
-	DEFINE_CMD_DESCRIPTOR (core, aft);
-	DEFINE_CMD_DESCRIPTOR (core, afv);
-	DEFINE_CMD_DESCRIPTOR (core, afvb);
-	DEFINE_CMD_DESCRIPTOR (core, afvr);
-	DEFINE_CMD_DESCRIPTOR (core, afvs);
-	DEFINE_CMD_DESCRIPTOR (core, ag);
-	DEFINE_CMD_DESCRIPTOR (core, age);
-	DEFINE_CMD_DESCRIPTOR (core, agn);
-	DEFINE_CMD_DESCRIPTOR (core, ah);
-	DEFINE_CMD_DESCRIPTOR (core, ahi);
-	DEFINE_CMD_DESCRIPTOR (core, ao);
-	DEFINE_CMD_DESCRIPTOR (core, ar);
-	DEFINE_CMD_DESCRIPTOR (core, ara);
-	DEFINE_CMD_DESCRIPTOR (core, arw);
-	DEFINE_CMD_DESCRIPTOR (core, as);
-	DEFINE_CMD_DESCRIPTOR (core, ax);
+static void cmd_anal_init(RCore *core, RCmdDesc *parent) {
+	DEFINE_CMD_DESCRIPTOR (core, parent, a);
```

Comment:
why not use core and "parent" inside the macro so you just need to specify the command name instead of repeating over and over. also this macro shhould be local to this file, so the name can be shorter